tooling/v8-snapshot/cache/darwin/snapshot-meta.json

```diff
@@ -871,7 +871,7 @@
     "./packages/server/lib/util/fs.ts",
     "./packages/server/lib/util/glob.js",
     "./packages/server/lib/util/proxy.ts",
-    "./packages/server/lib/util/resourceTypeAndCredentialManager.ts",
+    "./packages/server/lib/util/requestedWithAndCredentialManager.ts",
```

Comment:
This would normally be generated by my auto generator script so normally we shouldn't see these on individual pull requests, but I explicitly made this change locally just to make sure things were working in "prod mode" on my machine.